package/MDAnalysis/analysis/pca.py

```diff
@@ -114,8 +114,8 @@ class PCA(AnalysisBase):
         Take a pca_space and map it back onto the trajectory used to create it.
     """
 
-    def __init__(self, atomgroup, select='all', n_components=None,
-                 **kwargs):
+    def __init__(self, atomgroup, select='all', mean_free=True, reference=None,
```

Comment:
Please call the option `demean` this is used in scikit-learn and statsmodel to mean that the mean should be subtracted before any calculations are done.  I would like to use the same terminology to other python science packages to minimize surprise and friction for people using mdanalysis.